include/lcp-catlist.php

```diff
@@ -297,128 +309,134 @@ public function update_page($page) {
     $this->page = $page;
   }
 
-  public function get_posts_count(){
+  public function get_posts_count() {
     return $this->posts_count;
   }
 
-  public function get_number_posts(){
+  public function get_number_posts() {
     return $this->params['numberposts'];
   }
 
-  public function get_instance(){
+  public function get_instance() {
     return $this->instance;
   }
 
-  public function get_date_to_show($single){
-    if ($this->params['date'] == 'yes'):
+  public function get_date_to_show($single) {
+    if ($this->params['date'] == 'yes') {
       //by Verex, great idea!
       return ' ' . get_the_time($this->params['dateformat'], $single);
-    else:
+    } else {
       return null;
-    endif;
+    }
   }
 
-  public function get_modified_date_to_show($single){
-    if ($this->params['date_modified'] == 'yes'):
+  public function get_modified_date_to_show($single) {
+    if ($this->params['date_modified'] == 'yes') {
       return " " . get_the_modified_time($this->params['dateformat'], $single);
-    else:
+    } else {
       return null;
-    endif;
+    }
   }
 
   public function get_display_id($single) {
-    if (!empty($this->params['display_id']) && $this->params['display_id'] == 'yes'){
+    if (!empty($this->params['display_id']) && $this->params['display_id'] == 'yes') {
       $lcp_display_output .= $single->ID;
     }
   }
 
   public function get_no_posts_text() {
-    if ( ($this->get_posts_count() == 0) &&
-         ($this->params["no_posts_text"] != '') ) {
+    if (($this->get_posts_count() == 0) &&
+        ($this->params["no_posts_text"] != '')) {
       return $this->params["no_posts_text"];
     }
   }
 
-  public function get_content($single){
+  public function get_content($single) {
```

Comment:
I almost feel like we shouldn't even have this function, or the parameter. If people are needing to show the content, it's almost reimplementing the blog functionality... It's been there for too long to remove it now, but it's just a thought, I don't like having this feature :stuck_out_tongue_closed_eyes:

Reply:
I agree, WP is not intended to be used this way, a plugin should not run another main query to create a blog page. But I remember looking through support topics 2 years ago, many people use this 😆 I guess it's because WP expects you to customize blog page by editing the theme, but if you can't edit PHP you can use this plugin 😉 It's also a pain to maintain because output can differ based on theme and WP version.